src/main/java/com/github/crimsondawn45/fabricshieldlib/mixin/PlayerEntityMixin.java

```diff
@@ -79,62 +80,46 @@ private void disableShieldHead(boolean sprinting, CallbackInfo callbackInfo) {
 
         ShieldDisabledCallback.EVENT.invoker().disable(player, player.getActiveHand(), activeItemStack);
 
-        if (activeItem instanceof FabricShield) {
-            FabricShield shield = (FabricShield) activeItem;
+        if (activeItem instanceof FabricShield shield) {
 
             float f = 0.25F + (float) EnchantmentHelper.getEfficiency(player) * 0.05F;
             if (sprinting) {
                 f += 0.75F;
             }
 
             if (player.getRandom().nextFloat() < f) {
-                if (!FabricShieldLibConfig.universal_disable){
-                    player.getItemCooldownManager().set((Item) shield, shield.getCooldownTicks());
+                if (!FabricShieldLibConfig.universal_disable) {
+                    player.getItemCooldownManager().set((Item) shield, shield.getCoolDownTicks());
                     player.clearActiveItem();
                     player.world.sendEntityStatus(player, (byte) 30);
                     callbackInfo.cancel();
-                } else if (FabricShieldLibConfig.universal_disable){
-
-
-                    Optional<RegistryEntryList.Named<Item>> opt = Registries.ITEM.getEntryList(ConventionalItemTags.SHIELDS);
-                    List<Item> list = new ArrayList<>();
-                    if (opt.isPresent()){
-                        list = opt.get().stream().map(entry -> entry.value()).toList();
-                    }
-
-                    for(int amountOfShields = list.size(); amountOfShields > 0; amountOfShields--) {
-
-                        if (list.get(amountOfShields-1) instanceof ShieldItem){
-                            player.getItemCooldownManager().set(Items.SHIELD, 100);
-                        } else if(list.get(amountOfShields-1) instanceof FabricShield){
-                            player.getItemCooldownManager().set(list.get(amountOfShields-1), ((FabricShield)list.get(amountOfShields-1)).getCooldownTicks());
-                        }
-                        player.clearActiveItem();
-                        player.world.sendEntityStatus(player, (byte) 30);
-                    }
+                } else {
```

Comment:
keep the `else if` in case of `null`

Reply:
FabricShieldLibConfig.java
```java
@Entry
public static boolean universal_disable = false;
```
This is a primitive type and can not be null. If you want to use a nullable type for this case, you have to use "Boolean" instead of "boolean".